sdk/data/azappconfig/internal/generated/custom_client.go

```diff
@@ -26,3 +32,75 @@ func (a *AzureAppConfigurationClient) Pipeline() runtime.Pipeline {
 func (a *AzureAppConfigurationClient) Tracer() tracing.Tracer {
 	return a.internal.Tracer()
 }
+
+// copy of NewGetKeyValuesPager with slice of match conditions and other tweaks
+func (client *AzureAppConfigurationClient) NewGetKeyValuesPagerWithMatchConditions(matchConditions []azcore.MatchConditions, options *AzureAppConfigurationClientGetKeyValuesOptions) *runtime.Pager[AzureAppConfigurationClientGetKeyValuesResponse] {
```

Comment:
Whew... that name is a mouthful

Reply:
Fortunately this is all generated code that isn't exported.